src/sentry/tasks/reports.py

```diff
@@ -2,8 +2,10 @@
 
 import functools
 import itertools
+import json
```

Comment:
Use our `sentry.utils.json` instead. It's a bit more optimized and will be faster.